Isodose/Logic/vtkSlicerIsodoseModuleLogic.cxx

```diff
@@ -523,8 +666,25 @@ void vtkSlicerIsodoseModuleLogic::CreateIsodoseSurfaces(vtkMRMLIsodoseNode* para
     shNode->RemoveItem(isodoseFolderItemID, true, true);
   }
 
+  // Check if that absolute of relative values
+  bool relativeFlag = false;
+  vtkMRMLIsodoseNode::DoseUnitsType doseUnits = parameterNode->GetDoseUnits();
+  if (parameterNode->GetRelativeRepresentationFlag() 
+    && (doseUnits == vtkMRMLIsodoseNode::DoseUnitsType::GY
+    || doseUnits == vtkMRMLIsodoseNode::DoseUnitsType::UNKNOWN_UNITS))
+  {
+    relativeFlag = true;
+  }
+  else if (doseUnits == vtkMRMLIsodoseNode::DoseUnitsType::RELATIVE)
+  {
+    relativeFlag = true;
+  }
+  std::string isodoseName = (relativeFlag) ? 
+    vtkSlicerIsodoseModuleLogic::ISODOSE_RELATIVE_ROOT_HIERARCHY_NAME_POSTFIX :

```

Comment:
Why do we need to have two different postfixes? Can we use both at the same time?

Reply:
Two isolevels at the same time don't work quite right in hierarchy node. If both absolute and relative isolevels with the same postfix in hierarchy you can hide only one isolevel folder node, other can't be hidden. So if use only one postfix => only one isolevels folder node.